src/Aion/Transaction.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Transaction.h"
+
+#include "RLP.h"
+#include "../Ethereum/RLP.h"
+
+using namespace TW;
+using namespace TW::Aion;
+using boost::multiprecision::uint128_t;
+
+Data Transaction::encode() const noexcept {
+    auto encoded = Data();
+    append(encoded, Ethereum::RLP::encode(nonce));
+    append(encoded, Ethereum::RLP::encode(to.bytes));
+    append(encoded, Ethereum::RLP::encode(amount));
+    append(encoded, Ethereum::RLP::encode(payload));
+    append(encoded, Ethereum::RLP::encode(uint128_t(155157377101))); // Huge timestamp
```

Comment:
what's this hardcoded value?

Reply:
Aion-web3 takes [current timestamp * 1000](https://github.com/aionnetwork/aion_web3/blob/52453c82aa353f74f66f17f67d7d1f796c238d99/packages/web3-eth-accounts/src/index.js#L211) when building a transaction, if it's not provided by user. I just hardcoded some huge timestamp.